From 57d073c12e7bede29919117b0141df14015eb27f Mon Sep 17 00:00:00 2001
From: Tyler Hall <tylerwhall@gmail.com>
Date: Sun, 4 May 2014 20:06:43 -0400
Subject: [PATCH] python-config: Revert to using distutils.sysconfig

The newer sysconfig module shares some code with distutils.sysconfig, but the same modifications as in

12-distutils-prefix-is-inside-staging-area.patch makes distutils.sysconfig

affect the native runtime as well as cross building. Use the old, patched
implementation which returns paths in the staging directory and for the target,
as appropriate.

Upstream-Status: Inappropriate [Embedded Specific]

Signed-off-by: Tyler Hall <tylerwhall@gmail.com>
:

---
 Misc/python-config.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Misc/python-config.in b/Misc/python-config.in
index ebd99da..13e57ae 100644
--- a/Misc/python-config.in
+++ b/Misc/python-config.in
@@ -6,7 +6,7 @@
 import getopt
 import os
 import sys
-import sysconfig
+from distutils import sysconfig
 
 valid_opts = ['prefix', 'exec-prefix', 'includes', 'libs', 'cflags',
               'ldflags', 'extension-suffix', 'help', 'abiflags', 'configdir',
@@ -35,14 +35,14 @@ if '--help' in opt_flags:
 
 for opt in opt_flags:
     if opt == '--prefix':
-        print(getvar('prefix'))
+        print(sysconfig.PREFIX)
 
     elif opt == '--exec-prefix':
-        print(getvar('exec_prefix'))
+        print(sysconfig.EXEC_PREFIX)
 
     elif opt in ('--includes', '--cflags'):
-        flags = ['-I' + sysconfig.get_path('include'),
-                 '-I' + sysconfig.get_path('platinclude')]
+        flags = ['-I' + sysconfig.get_python_inc(),
+                 '-I' + sysconfig.get_python_inc(plat_specific=True)]
         if opt == '--cflags':
             flags.extend(getvar('CFLAGS').split())
         print(' '.join(flags))
